### DISCUSSION TITLE
SHA KEYS

### DISCUSSION THREAD

**Discussion by rahilhussain** [TheAlphamerc/flutter_twitter_clone#230] [Q&A] [answered]
unable to fetch sha-1 keys with command given in wiki on windows.If any one get those help me.

he term 'keytool' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path 
was included, verify that the path is correct and try again.
At line:1 char:1
+ keytool -list -v -keystore "\.android\debug.keystore" -alias androidd ...
+ ~~~~~~~
    + CategoryInfo          : ObjectNotFound: (keytool:Str

**Comment by TheAlphamerc** [comment]
@rahilhussain Its looks like you need to install keytool first before running this command.

**Comment by rahilhussain** [comment, marked as answer]
@TheAlphamerc i solved it by using gradlew. Thanks anyways 😊
